apps/aecore/src/aec_peer_messages.erl

```diff
@@ -88,7 +109,7 @@ serialize(peers, Peers, Vsn = ?VSN_1) ->
     [ serialize(peer, Peer, Vsn) || Peer <- Peers ];
 serialize(peer, Peer, ?VSN_1) ->
     %% Don't pollute the encoding with lots of Vsn-fields...
-    Template = serialization_template(peer, ?PING_VSN),
+    Template = serialization_template(peer, ?VSN_1),
```

Comment:
```suggestion
    Template = serialization_template(peer, ?PING_VSN_1),
```

Reply:
Thanks, there is actually PEER_VSN already defined in aec_peer_messages.hrl. I'll change it to be more consistent. The ping serialisation already uses PING_VSN_2 and PING_VSN_1.